.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Before you get started
+- [ ] 🙋‍♀️ Create an issue to discuss what you are doing to do!
+
+---
+
+## Description
+- Closes #issue_number
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 
+
+## Type of change
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How has this been tested?
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test configuration**:
+* Operating system:
+* Other relevant information:
+
+## Checklist
```

Comment:
I found the first (technical) part of the [ESMValTool checklist](https://github.com/ESMValGroup/ESMValTool/blob/main/.github/pull_request_template.md) somewhat more convenient. The items are shorter and there are links to the subtopics. We could add sections in our README file and describe our detailed requirements there and add links in pull_request_template.md

Reply:
Agreed. Once we have contribution guidelines we can link to them more specifically. Would you like to set these up first and then introduce these templates?

Reply:
For me it is okay to change that once the contribution guidelines are ready. Meanwhile, the current checklist (without) links can be used. Can you please open a new issue for the change with links to the contribution guidelines?